web/packages/teleport/src/types.ts

```diff
@@ -118,3 +164,10 @@ export type LockedFeatures = {
   premiumSupport: boolean;
   trustedDevices: boolean;
 };
+
+// RecommendFeature is used for recommending features if its usage status is zero.
+export type RecommendFeature = {
+  TrustedDevices: RecommendationStatus;
+};
+
+export type RecommendationStatus = 'NOTIFY' | 'DONE';
```

Comment:
Can this be an enum?

Reply:
Updated this to enum in https://github.com/gravitational/teleport/pull/29757/commits/007fcdb19d5d45a2e664b0aa9208dbe9b446e9ee